tests/Unit/ApiTest.php

```diff
@@ -540,8 +540,9 @@ public function test_get_product_facebook_ids_creates_get_ids_request() {
 		$response = function( $result, $parsed_args, $url ) use ( $facebook_product_catalog_id, $facebook_product_retailer_id ) {
 			$this->assertEquals( 'GET', $parsed_args['method'] );
 
-			$path = "catalog:{$facebook_product_catalog_id}:" . base64_encode( $facebook_product_retailer_id );
-			$path = "/{$path}/?fields=id,product_group{id}";
+			$filter = urlencode('{"retailer_id":{"eq":"' . $facebook_product_retailer_id . '"}}');
+			$fields = urlencode('id,product_group{id}');
+			$path = "/{$facebook_product_catalog_id}/products?filter={$filter}&fields={$fields}";
 
 			$this->assertEquals( "{$this->endpoint}{$this->version}{$path}", $url );
 			return [
```

Comment:
Just making sure: is the api response structure in production still matches one in the test here after you updated request?